src/internal/events.ts

```diff
@@ -12,6 +12,10 @@ export const AccountCreatedEventStruct = object({
      * New account object.
      */
     account: KeyringAccountStruct,
+    /**
+     * Display the add account confirmation in UI.
```

Comment:
Probably we should word it differently to say that it instructs, or asks, MetaMask to display or hide the confirmation, but it's not garanteed that MetaMask will honor this flag.

Reply:
Also addressed here [c125539](https://github.com/MetaMask/keyring-api/pull/307/commits/c12553984bde91b656adbdc51a0b5471bc813bf2)